WooCommerce/Classes/Analytics/WooAnalyticsEvent.swift

```diff
@@ -605,6 +605,54 @@ extension WooAnalyticsEvent {
     }
 }
 
+// MARK: - InPersonPayments Feedback Banner
+extension WooAnalyticsEvent {
+    enum InPersonPaymentsFeedbackBanner {
+        /// Possible sources for the Feedback Banner
+        ///
+        enum Source: String {
+            static let key = "source"
```

Comment:
I think this `key` isn't used anywhere.

Reply:
Oops, I duplicated the value. Fixed on 30c18269d2d7a5acbd435ae6a2bcd2497ff4938c